### PR TITLE
Shadowling Update IV

### PR DESCRIPTION
**Pull request by ghost**
- Fixes shadowlings being unable to enthrall up to 5 before hatching
- Black Recuperation now has a 1 minute cooldown (down from 5)
- Empowered thralls have been added
- Empowered thralls look similar to shadowlings in addition to slowly dying in light and healing in the dark, as well as possessing better abilities
- Black Recuperation can also be used to empower living thralls
- Guise no longer requires it to be dark
- Guise now sets the user to 10 alpha (down from 20)
- Torches are now as bright as flashlights, due to their mass-producability. They also no longer fit on belts.
- Shadowlings can now adjust their night vision from 0 to 8 tiles. This ability is attached to their eyes (technically, glasses). Demo: https://dl.dropboxusercontent.com/u/116194515/slingvision.gif

### CODE REVIEW

File: code/modules/surgery/dethrall.dm [hunk 1]
```diff
@@ -34,6 +54,20 @@
 			else
 				user.visible_message("<span class='warning'>[user] screws up!</span>")
 			return 0
+		if(target.dna.species.id == "l_shadowling")
```
Comment (Xhuis) [inline code/modules/surgery/dethrall.dm]: This copypasta doesn't exist on my local repo and I'm not sure why

**Comment by PKPenguin321** [comment]
hey xhuis, this is just a request and you don't really *have* to address it but it would be really really nice if you could tinker with the wiki to accommodate for changes, it's wildly out of date and very likely one of the big causes of unrobust shadowlings

and if this PR doesn't get merged for over two days then make a separate PR for
>Fixes shadowlings being unable to enthrall up to 5 before hatching

it's fairly critical and pretty much every new shadowling gets cucked by it

**Comment by Xhuis** [comment]
I was actually updating the wiki but it went down

**Comment by GunHog** [comment]
Nightvision toggle, please?

**Comment by Xhuis** [comment]
Customizable night vision is here! Read OP for more details.

**Comment by Xhuis** [comment]
Squashed commits as well

**Comment by PKPenguin321** [comment]
>I was actually updating the wiki but it went down. Every time I try to change the wiki it seems that I either can't get on it or it just goes down.

I suppose I could *try* to rewrite it for you but it would be nice if you could read over it real fast and spitball things that need to be changed in a pastebin or something
I would do it entirely myself but I don't know how shadowlings work to that extent yet

**Comment by Xhuis** [comment]
Wiki is updated

**Comment by Robustin** [comment]
I'm opposed to re-adding pre-hatched enthralling.

This always, always, ALWAYS encouraged a completely cancerous playstyle where idiots would try and get the ball rolling with a stunprod, cable cuffs, etc. The victim always screams and the ling either gets caught immediately or a short while later when sec/borgs sweep the area. A handful of players can make it work but that doesn't balance out the other 95% who are playing Greyshirt: Team Edition. 

Leave the stupid kidnapping tactics to the thralls, thats why they exist. 

It's a huge negative on Shadowling mode when youve got 2-3 out of the 4 shadowlings get caught and executed before they even hatch while simultaneously making it infinitely harder for the hatched players to stay hidden/alive while doing their conversions. It's boring as security too when the "enemy" you're facing is just a stunprod assistant who somehow thinks they won't get executed after kidnapping someone and trying to enthrall them.

Don't encourage this playstyle, shadowlings is better when you actually have to fight shadowlings and not idiots with stunprods.

**Comment by Xhuis** [comment]
Pre-hatch enthralling has been in since the second update, just nonfunctional due to bugs. You can only enthrall a small amount of people before you hatch, which I felt made it balanced enough to warrant keeping the feature in.

**Comment by PKPenguin321** [comment]
yeah, when i go into full shadowling form I'd like to have at least one thrall by my side,  pre-hatch enthralling isn't for "le convert people as greyshirt" but more for "try to gather a couple people so you don't die instantly to the first guy with a flare"

**Comment by Xhuis** [comment]
Again, DNM tag can be removed

**Pull request merged**